azure-kusto-data/azure/kusto/data/client_base.py

```diff
@@ -58,6 +61,11 @@ def __init__(self, kcsb: Union[KustoConnectionStringBuilder, str], is_async):
         self.client_details = self._kcsb.client_details
         self._is_closed: bool = False
 
+        self.default_database = self._kcsb.initial_catalog
+
+    def get_database(self, database_name: str) -> str:
```

Comment:
This can be a property as in KCSB

Reply:
There is a "inital_catalog" property in kcsb, that's only a small helper for client
It should probably be private too